README.md

```diff
@@ -72,6 +72,49 @@ Custom registries are not currently supported
 ([#160](https://github.com/obi1kenobi/cargo-semver-check/issues/160)), so crates published on
 registries other than crates.io should use one of the other approaches of generating the baseline.
 
+### What features does `cargo-semver-checks` enable in the tested crates?
+
+By default, crates are built with all features excluding heuristically selected features that usually break semver. More precisely, features named `unstable`, `nightly`, `bench`, `no_std` or starting with `__` are disabled.
```

Comment:
Try to avoid linker phrases like "more precisely." I know we're all taught in school that we should connect our sentences together — I'm still working on getting rid of "therefore" and "also" in my own writing. But these are all just fluff, and make the writing less concise without helping the reader get the point.

Better to just get straight to the point.

Reply:
How about something like:
```suggestion
By default, checking is done on all features except features named `unstable`, `nightly`, `bench`, `no_std`, or ones with prefix `_`, `unstable-`, or `unstable_`, as such names are commonly used for private or unstable features.
```
It's roughly the same length. I've also updated it for the heuristic tweaks suggested by Ed Page.

Reply:
Looks good!